src/components/Tag.js

```diff
@@ -63,8 +73,21 @@ const Title = styled(TextElement).attrs(({ color, theme: { colors } }) => ({
   marginBottom: 1,
 });
 
-const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
+const Tag = ({
+  color,
+  disableMenu,
+  slug,
+  text,
```

Comment:
I don't understand where this `text` can come from? 

It is not passed in `UniqueTokenAttributes` as an additional prop. Also, this is not in `MappedTrait` or `UniqueAssetTrait`. I suspect this might be a part of `UniqueAssetTrait` that was not mentioned in docs. OS docs clearly said the docs are not exhaustive about the potential fields of trait.

Reply:
`text` is a mapped `value` like parsed date etc. the thing that will be displayed `originalValue` is the value we will use for OpenSea link or to open in the web browser. Can I make it less confusing in some way? The name `text` seems to be reasonable since it is displayed, I could use `textValue` `originalValue` if you have something else in mind let me know.
Below some explanation

In `UniqueTokenAttributeItem.tsx` file you can find below
```
<Tag
    color={color}
    disableMenu={disableMenu}
    lowercase={lowercase}
    maxValue={maxValue}
    originalValue={originalValue}
    slug={slug}
    // here is the prop passed
    text={value}
    title={type}
  />
  ```
  
  And then in `UniqueTokenAttributes.tsx` file
  
  ```
  <Inline space="10px">
      {sortedTraits.map(item => (
        <UniqueTokenAttributeItem
          key={`${item.trait_type}${item.originalValue}`}
          // here is the complete sorted and mapped trait item passed, it includes value that is then passed to a text Tag prop
          {...item}
        />
      ))}
    </Inline>
    ```

Reply:
ok